### PR TITLE
Reconcile DRIVER/ashlar split

### PR DESCRIPTION
**Pull request by kshepard**
* Fixes problem where incorrect wsgi config was being used
 * Consolidates gunicorn services (now uses just the default from the azavea.gunicorn role)
 * Changes all ashlar references to driver for things such as file names, users, service names, db setup

You will most likely want to destroy your VMs and reprovision after this is merged.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -89,7 +89,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.define "database" do |database|
     database.vm.hostname = "database"
-    database.hostmanager.aliases = %w(database.service.ashlar.internal)
+    database.hostmanager.aliases = %w(database.service.driver.internal)
```
Comment (CloudNiner) [inline Vagrantfile]: theres a hostmanager config for `app.service.ashlar.internal` in this file, might want to switch that one to driver

Comment (kshepard) [inline Vagrantfile]: Good call. I'm rebasing your Travis changes too -- need to change references for the settings defaults.

**Comment by CloudNiner** [comment]
+1 was able to reprovision and get an app.

**Comment by CloudNiner** [comment]
I'll spend some more time trying to figure out npm travis failures.

**Pull request merged**